### ISSUE TITLE
How to switch to uglyURLs=true for html files  

### ISSUE THREAD

**Issue by sagayd** [McShelby/hugo-theme-relearn#496]
I turned on uglyURLs=true for my website. However my site still generates `<topic-dir>/index.print.html` and `<topic-dir>/index.xml`. Can  I change the print file to be at blogs root like `blogs/<topic-name>.print.html`  (for PRINT formati) and  `blogs/<topic-name>.xml` (for RSS format). Thanks

**Comment by sagayd** [comment]
My config.toml file has the below content:
`[mediaTypes."text/print"]
suffixes = ["print.html"]

[outputFormats]
  [outputFormats.PRINT]
    name= "PRINT"
    isHTML = true
    mediaType = 'text/print'
    #noUgly = false #true > pretty, false => uglyURL

[outputs]
  page = ["HTML", "PRINT"]
`

However it does not generate uglyURL for Print format. Any fixes?

**Comment by McShelby** [comment]
Please give a complete config.toml of your project, the relative quallified name and path of your `.md` file and the URL you want to achieve for your file or even better - give a link to your projects repository

**Comment by sagayd** [comment]
Hi @McShelby 

Here is my public repo (for now) https://github.com/sagayd/docs-ad-hugo.git. I turned on print and RSS outputs, Ideally I expect the file output with `<article>.extension`, for a reason. 

I have also raised discussion topic with Hugo team, but after testing they feel its little complicated with the selected theme.
Here is the discussion link: https://discourse.gohugo.io/t/how-to-stop-generating-index-html-for-article/43138
And their, printable path testing code: https://github.com/jmooring/hugo-testing/blob/hugo-forum-topic-43138/

**Comment by sagayd** [comment]
Hi @McShelby, Any updates please

**Comment by McShelby** [comment]
As time is sparse, this will take a while to look into. Sorry for not having a quicker response.

**Comment by sagayd** [comment]
Hi @McShelby, I have been waiting for your update. Any idea when can I get solution?

**Comment by McShelby** [comment]
tldr; currently you can't redefine file names for PRINT nor RSS.

While this is usually possible with other themes, this theme doesn't support it.

If this will ever be supported, please remember that jmooring's solution is using an unknown mime type which will also show up in the generated pages if not hacked around.
